PC02_Graffiti_COOPER-SANSONE.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Andrew Cooper-Sansone (9/4/2020)
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
```

Comment:
Add a comment indicating a "Screen Setup Section"
```suggestion

#Screen Setup Section:
panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
panel.bgcolor("lightsteelblue1")
panel.bgpic(image)
```